bin/ansible-connection

```diff
@@ -291,9 +291,14 @@ def main():
         sys.stderr.write(traceback.format_exc())
         sys.exit("FAIL: %s" % e)
 
+    # disable logging on stdout, to avoid spurious debug logs to
+    # be outputted on stdout.
+    C.DEFAULT_LOG_ONLY = True
```

Comment:
The [Network troubleshooting](https://docs.ansible.com/ansible/network_debug_troubleshooting.html#unable-to-open-shell) docs mention enabling debug logs to troubleshoot connection related issues. This will impact all the Network platforms using persistent connection framework.